src/content/core/using-okteto-cli.mdx

```diff
@@ -5,143 +5,64 @@ sidebar_label: Using the Okteto CLI
+If you're using a Docker Compose file, then Okteto CLI can directly launch a development environment for your application without generating an Okteto manifest. However, you can still choose to create the Okteto manifest if you want more control over the configuration of your environment.
 
-#### Built-in Environment Variables for Dependencies
+> Note: You can use the `-f` flag on any Okteto CLI command to explicitly define the location of an Okteto manifest.
 
-You can refer to the variables defined by your [dependencies](reference/okteto-manifest.mdx/#dependencies-object-optional) using the following environment variables:
-- `OKTETO_DEPENDENCY_<dependency>_VARIABLE_<variable>`: the value of a variable of a dependency.
+## Development environments
 
-For example, if you have the following dependency in your Okteto Manifest:
+When you deploy an application to Okteto we create a development environment for you using your deployment manifest's instructions. A developer environment consists of the various services and components of your application and are represented within the Okteto UI like so:
 
-```yaml
-dependencies:
-  database:
-    repository: https://github.com/okteto/my-database
-    variables:
-      USERNAME: okteto
-    wait: true
-```
+<p align="center">
+  <Image
+    src={require("@site/static/img/okteto-ui-dev-env.png").default}
+    alt="UI showing the movies app dev environment"
+    width="350"
+  />
+</p>
 
-and the repo https://github.com/okteto/my-database has the following Okteto Manifest:
+## Development containers
 
-```yaml
-deploy:
-  - echo "PASSWORD=1234" >> $OKTETO_ENV
-```
-
-You can refer to the variables of https://github.com/okteto/my-database in the `deploy` section of your Okteto Manifest like this:
-
-```yaml
-dependencies:
-  database:
-    repository: https://github.com/okteto/my-database
-    variables:
-      USERNAME: okteto
-    wait: true
-deploy:
-  - echo ${OKTETO_DEPENDENCY_DATABASE_VARIABLE_USERNAME}
-  - echo ${OKTETO_DEPENDENCY_DATABASE_VARIABLE_PASSWORD}
-```
-
-Note that `USERNAME` has an static value, and `PASSWORD` is dynamically generated.
+In the Okteto manifest you can define the [`dev` section](reference/okteto-manifest.mdx#dev-object-optional) which specifically informs Okteto about how to build and deploy a [development container](deploy/development-environments.mdx) for each service or component you'd like to work on. This feature enables you to make local changes to your deployed application and see those changes reflected immediately within the remote development container in Okteto. The Okteto CLI creates a bridge between your local machine and your Okteto development container so you can continue coding in your favorite IDE and see changes live on the deployed version of your application as soon as you hit save.
 
 ### Built-in Tools When Deploying To Okteto
```

Comment:
I don't think this section belongs here. This are tools available in your `deploy` commands

Reply:
@pchico83 I'm not sure I understand. I wanted to convey here that "dev containers exist and are how we provide part of the Okteto experience".

I'm not sure what "tools available in your `deploy` commands" means in this context.

If this section were re-written would it be a better fit or do you think we should remove it entirely?

Reply:
Removing the section for now and will add it back in here or wherever we deem it appropriate in a follow-on PR 👍